google/cloud/forseti/services/server.py

```diff
@@ -403,6 +405,9 @@ def serve(endpoint,
     # Configuring log level for the application
     logger.set_logger_level_from_config(log_level)
 
+    if enable_console_log:
```

Comment:
Can you just move this higher up in the method, so that we can capture all the logging to the console?

Reply:
Updated.